### PR TITLE
2020-01-13

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2020-01-15-weak-passwords.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+type: post
+published: true
+status: publish
+categories: [Tech]
+image:
+image2:
+author: Valentino Urbano
+---
+
+When I started using 1Password I manually imported the logins I remembered and changed the passwords foe the most important ones. The others were left with relatively weak passwords, but if they got compromised nothing bad would happen. A few days ago I became curious in finding those account and taking a look at them after almost a decade.
```
Comment (MsSlacker) [inline _posts/2020-01-15-weak-passwords.md]: foe - > for

Comment (MsSlacker) [inline _posts/2020-01-15-weak-passwords.md]: The app doesn't let you select text and copy... So lame.

Rewrite the last sentence here.

File: _posts/2020-01-15-weak-passwords.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## How To Show Weak passwords in 1Password
+
+Open 1Password and click the “Traditional” layout button to see all your data in columns. The button is just left of the search field at the top of the window or you can select it from the View > Layout menu.
+Make sure View > Columns > Password Strength is enabled.
+Now click the Password Strength column header to sort all your Logins by the strength of their passwords.
+
+## Results
+
+The astounding results was finding in the list, apart from the expected old passwords, also passwords that I was forced to create as weak because of restrictions of that specific website password policy. More than that a few of these weak passwords were from banks. Those bank accounts are also protected with two factor authentication so that would not be such of a big concern, but it is telling at how banks are still behind in this area.
```
Comment (MsSlacker) [inline _posts/2020-01-15-weak-passwords.md]: I was astounded to find or something like that

File: _posts/2020-01-15-weak-passwords.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+
```
Comment (MsSlacker) [inline _posts/2020-01-15-weak-passwords.md:27]: Bit short....

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2020-01-15-weak-passwords.md (hunk 3 above)
Comment (MsSlacker) [reply _posts/2020-01-15-weak-passwords.md:27]: Zzz? I have been coming here for like a week and there's nothing to do...

Comment (valeIT) [reply _posts/2020-01-15-weak-passwords.md:27]: If you have any idea on something to add let me know =) But yeah I generally tend to write short stuff

File: _posts/2020-01-15-weak-passwords.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ image2:
 author: Valentino Urbano
 ---
 
-When I started using 1Password I manually imported the logins I remembered and changed the passwords foe the most important ones. The others were left with relatively weak passwords, but if they got compromised nothing bad would happen. A few days ago I became curious in finding those account and taking a look at them after almost a decade.
+When I started using 1Password I manually imported the logins I remembered and changed the passwords for the most important ones. The others were left with relatively weak passwords, but if they got compromised nothing bad would happen. A few days ago I became curious  and started trying to find out which account were part of this group after almost a decade.
```
Comment (MsSlacker) [inline _posts/2020-01-15-weak-passwords.md]: which account - > ich of the accounts

**Review by MsSlacker — CHANGES_REQUESTED** [review]

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**